extensions/ql-vscode/src/common/vscode/telemetry.ts

```diff
@@ -297,3 +297,21 @@ export async function initializeTelemetry(
   ctx.subscriptions.push(telemetryListener);
   return telemetryListener;
 }
+
+function isGlobalTelemetryEnabled(): boolean {
+  // If "enableTelemetry" is set to false, no telemetry will be sent regardless of the value of "telemetryLevel"
+  const enableTelemetry: boolean | undefined =
+    GLOBAL_ENABLE_TELEMETRY.getValue();
+  if (enableTelemetry === false) {
+    return false;
+  }
+
+  // If a value for "telemetry.telemetryLevel" is provided, then use that
+  const telemetryLevel: string | undefined = GLOBAL_TELEMETRY_LEVEL.getValue();
+  if (telemetryLevel !== undefined) {
+    return telemetryLevel === "error" || telemetryLevel === "on";
```

Comment:
These are the levels that I see:

<img width="487" alt="Window_and_Settings_—_codeql-action" src="https://github.com/github/vscode-codeql/assets/363559/91a2b071-09a2-4522-8433-6b092ec5f45b">

So, I think you want to change `on` to `all`.

Reply:
You're right. I was originally reading https://code.visualstudio.com/updates/v1_61#_telemetry-settings but that's a changelog entry and is apparently not outdated. In my local vscode and some other docs I also see what you see.